src/PageLayout/Notification/NotificationPopup.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import truckG from '../../icons/green_truck.svg';
+import { NotificationType, AccountType } from '../../Enums';
+import './Popup.css';
+
+class NotificationPopup extends Component {
+    render() {
+        return (
+            <div>
+                {
+                    this.props.notificationType === NotificationType.RECURRING_PICKUP_REQUEST &&
+                    (this.props.account === AccountType.RECEIVING_AGENCY || this.props.account === AccountType.DELIVERER_GROUP) ?
+                        <div className="notification-wrapper recurring arrow-up">
+                            <img className="popup-icon" src={truckG} alt="icon"/><h1 className="hover">Recurring pickup requested</h1>
+                            <div className="notification-buttons notification">
+                                <button onClick={this.props.clickNotification} className="claim notification" type="button">View</button>
+                            </div>
+                        </div>
+                        :
+                        null
+                }
+                {
+                    this.props.notificationType === NotificationType.RECURRING_PICKUP_CONFIRMED &&
+                    this.props.account === AccountType.DONATING_AGENCY_MEMBER ?
+                        <span className="notification-wrapper recurring">
+                            <img className="popup-icon" src={truckG} alt="icon"/><h1 className="hover">A recurring pickup has been claimed</h1>
+                            <div className="notification-buttons notification">
+                                <button onClick={this.props.clickNotification} className="claim notification" type="button">View</button>
+                            </div>
+                        </span>
+                        :
+                        null
+                }
```

Comment:
- we dont' actually need to check for the account type at all in here. This is because each account would only have the notification if it exists in the db in their account object. And the notification objects are added to their account object only if they should have it. So here, we can just always display whatever notification they have according to the db.
- Instead of having different cases in `render` here, what I was thinking is that you could add another file `NotificationMap.js` that looks like:
```
import { NotificationType } from '../../Enums.js';

export default const NotificationMap = {
    NotificationType.RECURRING_PICKUP_REQUEST: {
        iconSrc: 'truckG',
        msg: 'Recurring pickup requested'
    },
    NotificationType. RECURRING_PICKUP_CONFIRMED: {
        iconSrc: 'truckG',
        msg: 'A recurring pickup has been claimed'
    },
    ...
};
```
And then in here, all you would have is one case:
```
let info = NotificationMap[this.props.notificationType];
...
<div .....>
    <img ..... src={info.iconSrc} ... />
    <h1 ...>{info.msg}</h1>
    ...
</div>
```
Let me know if this makes sense. I might be overlooking something too. And feel free to change any naming.